css/main.css

```diff
@@ -1,3 +1,373 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+header {
+  padding-top: 50px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 150px;
+  background-color: white;
+}
+
+.draw-title {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.main-article {
+  margin-left: 10px;
+  font-size: 2rem;
+  font-weight: 700;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.sec-article {
+  margin-left: 5px;
+  font-size: 2rem;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.the-nav {
+  margin-right: 10%;
+  margin-left: 22%;
+}
+
+.blue-square {
+  background-color: blue;
+  min-width: 20px;
+  height: 20px;
+  margin-left: 5%;
+  margin-top: 15px;
+}
+
+.nav-list {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  padding: 2%;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size: 1.2rem;
+}
+
+.burger-menu {
+  display: none;
+}
+
+a:link {
+  text-decoration: none;
+  color: black;
+}
+
+a:visited {
+  color: lightblue;
+}
+
+a:hover {
+  color: blue;
+  cursor: pointer;
+}
+
+a:active {
+  text-decoration: underline;
+  color: darkblue;
+}
+
+main {
+  margin-top: 200px;
+}
+
+.first-row {
+  height: 150px;
+}
+
+.gray-div {
+  background: rgb(216, 212, 190);
+}
+
+.light-gray-div {
+  background: rgb(240, 237, 224);
+}
+
+.photo-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.my-photo {
+  width: 300px;
+  border-radius: 50%;
+}
+
+.small-seperator {
+  width: 80px;
+  border: 2px solid darkblue;
+}
+
+.small-photos {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding: 5%;
+  padding-left: 20%;
+  padding-right: 20%;
+  background-color: white;
+  width: 100%;
+  margin-bottom: 0px;
+  margin-top: 45px;
+}
+
+.small-photo {
+  width: 25px;
+}
+
+.Intro-div {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-start;
+}
+
+.welcome-article {
+  margin-left: 10px;
+  font-size: 6rem;
+  font-weight: 700;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.who-me {
+  margin-left: 10px;
+  font-size: 1.5rem;
+  font-weight: 300;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.div-buttons {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 3%;
+  margin-top: 45px;
+}
+
+.res-btn {
+  background-color: rgb(58, 76, 240);
+  color: white;
+  font-weight: 300;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  width: 150px;
+  border-radius: 15px;
+}
+
+.proj-btn {
+  background-color: white;
+  color: rgb(58, 76, 240);
+  font-weight: 300;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  width: 150px;
+  border-radius: 15px;
+  margin-left: 20px;
+}
+
+.res-btn:hover {
+  background-color: white;
+  color: rgb(58, 76, 240);
+}
```

Comment:
Very neat 👍 
Avoid using `width` for buttons in case the text inside changes - you want your button to adapt to its content. Instead, just add `padding` and the button will change size according to the text.